src/DynamicMvvm.Abstractions/DynamicMvvm.Abstractions.csproj

```diff
@@ -2,6 +2,7 @@
 
 	<PropertyGroup>
 		<TargetFrameworks>netstandard2.0</TargetFrameworks>
+		<LangVersion>10.0</LangVersion>
```

Comment:
Why would you need this?

Reply:
Could not compile with 17.2.X I believe with dotnet 6 :)

Reply:
But that library is .net std 2.0 and doesn't use C# 10...

Reply:
This was a mistake, this line is only necessary in the WinUI head